node/cli/Cargo.toml

```diff
@@ -69,7 +69,7 @@ vara-native = [
 	"service/vara-native",
 	"gear-runtime-test-cli?/vara-native",
 	"vara-runtime",
-	"gear-program/vara"
+	"gear-program/gear"
```

Comment:
sorry didn't get that change. `gear-program` declares its own `vara` feature and there it is used for `vara-native` that looks logical.